mentors/javascript/a.json

```diff
@@ -45,6 +45,6 @@
     "link_text": "https://aibrahim3546.github.io/",
     "link_url": "https://aibrahim3546.github.io/",
     "avatar_url": null,
-    "bio": "I Javascript developer, ReactJs, NodeJs, React Native."
+    "bio": "I am a fan of javascript. I have been working with ReactJs for a year and yeah I am really good at ReactJs. Also Good at NodeJs and React Native"
```

Comment:
Nice!  Welcome to Exercism as a javascript mentor!

Reply:
Thanks mate